src/threading/thread.cpp

```diff
@@ -311,7 +311,7 @@ bool Thread::bindToProcessor(unsigned int proc_number)
 
 bool Thread::setPriority(int prio)
 {
-#if USE_WIN_THREADS
+#ifdef _MSC_VER
```

Comment:
If I remember correctly because it couldn't cast the return value of `getThreadHandle()` to a `HANDLE *` (`VOID *`). MSVC seems to have either a overload or an implicit cast.